test/integration/api.js

```diff
@@ -262,7 +262,15 @@ describe('api v.1', () => {
                 user_id: user.get('id'),
                 text: `This is a Post #${i}`
               });
-              return post.save({fully_published_at: (new Date(Date.now() - 50000 + i*1000)).toJSON()}, {method: 'insert'});
+
+              const dateJson = new Date(Date.now() - 50000 + i*1000).toJSON();
+              let defaultAttr = { fully_published_at: dateJson, created_at: dateJson };
```

Comment:
it's ok to use const here too, as you never reassign the variable (no need to change it, just saying)
const != immutable